.github/workflows/main.yml

```
@@ -74,6 +74,23 @@ jobs:
           RESOURCE: .nais/preprod.yml
           VAR: image=${{ needs.build-and-push.outputs.image }}
 
+  deploy-qa-gcp:
+    name: Deploy to preprod
```

```suggestion
    name: Deploy to dev-gcp
```

Det er litt vanskelig å skille disse når de heter det samme :sweat_smile: